src/main/java/duke/TaskManager.java

```diff
@@ -0,0 +1,103 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+import duke.tasks.*;
+
+public class TaskManager {
+    public static final String END_OF_LINE = "\n____________________";
+    private static List<Task> tasks = new ArrayList<Task>();
+
+    public void addNewTodo(String taskName, boolean toPrint) {
+        tasks.add(new Todos(taskName));
+        if (toPrint) {
+            System.out.println("Added new todo task: " + taskName + "\n " + tasks.get(tasks.size() - 1).toString()
+                    + "\nYou have " + tasks.size() + " tasks in the list." + tasks);
+        }
+    }
+
+    public void addNewDeadline(String taskName, String toBeDoneBy, boolean toPrint) {
+        tasks.add(new Deadlines(taskName, toBeDoneBy));
+        if (toPrint) {
+            System.out.println("Added new deadline task: " + taskName + "\n " + tasks.get(tasks.size() - 1).toString()
+                    + "\nYou have " + tasks.size() + " tasks in the list." + END_OF_LINE);
+        }
+    }
+
+    public void addNewEvent(String taskName, String happeningAt, boolean toPrint) {
+        tasks.add(new Events(taskName, happeningAt));
+        if (toPrint) {
+            System.out.println("Added new event task: " + taskName + "\n " + tasks.get(tasks.size() - 1).toString()
+                    + "\nYou have " + tasks.size() + " tasks in the list." + END_OF_LINE);
+        }
+    }
+
+    public void listTasks() {
+        System.out.println("Here is your list of tasks.");
+        for (int i = 0; i < tasks.size() - 1; i++) {
+            System.out.println((i + 1) + ". " + tasks.get(i).toString());
+        }
+        if (tasks.size() > 0) {
+            System.out.println((tasks.size()) + ". " + tasks.get(tasks.size() - 1).toString() + END_OF_LINE);
+        }
+    }
+
+    public void markTasks(boolean toMark, int taskIndex, boolean toPrint) throws DukeException.IllegalMarkTargetException {
```

Comment:
Consider renaming the booleans to `isMark` and `isPrint` in accordance to the naming convention within the coding standard.